.github/workflows/ci.yml

```diff
@@ -1,43 +1,34 @@
-name: Test
+name: Ruby
 
 on:
   push:
     branches: [main, master] # TODO: rename and get rid of 'master'
-  # Triggers the workflow on pull request events.
+    paths-ignore:
+      - '**/*.md'
   pull_request:
-    types: [opened, reopened, synchronize]
-  # Allows you to run this workflow manually from the Actions tab
+    branches: [main, master] # TODO: rename and get rid of 'master'
+    paths-ignore:
+      - '**/*.md'
```

Comment:
Minor refactor, you don't want CI running for doc changes do you? Of course not.